News/Entities/UserEntity.cs

```diff
@@ -1,20 +1,20 @@
-﻿using System.ComponentModel.DataAnnotations.Schema;
-
-namespace News.Entities;
+﻿namespace News.Entities;
 
 public class UserEntity
 {
     public Guid Id { get; set; }
 
-    public string? Username { get; set; }
-
-    public Guid PasswordId { get; set; }
-
-    public PasswordEntity? Password { get; set; }
+    public string? FirstName { get; set; }
```

Comment:
We need to discuss how the name will be used—whether it will be a full name or possibly just a first name.

Reply:
I guess now its nevermind. We can extand user information later